vcbuild.bat

```diff
@@ -246,6 +248,17 @@ ssh -F %SSHCONFIG% %STAGINGSERVER% "touch nodejs/%DISTTYPEDIR%/v%FULLVERSION%/no
 :run
 @rem Run tests if requested.
 
+:tar-headers
+@rem build headers package
+if "%tar_headers%"=="" goto build-node-weak
+set HEADERS_ONLY=1
+set TARNAME=node-%FULLVERSION%
+python tools\install.py install %TARNAME% ""
+7z a %TARNAME%-headers.tar %TARNAME%
+rmdir /s /q %TARNAME%
+7z a %TARNAME%-headers.tar.gz %TARNAME%-headers.tar
+del %TARNAME%-headers.tar
```

Comment:
A `zip` and `7z` would be better (similar to the `gz` and `7z` on unix). We did some work to find the best flags for `7z`: https://github.com/janeasystems/node/blob/36b525a78119a1fc3af9a6934762b4cee0f78a5c/vcbuild.bat#L243-L251